php_memcached.stub.php

```diff
@@ -17,8 +17,8 @@ public function get(string $key, callable $cache_cb=NULL, int $get_flags=0): mix
 	public function getByKey(string $server_key, string $key, callable $cache_cb=NULL, int $get_flags=0): mixed {}
 	public function getMulti(array $keys, int $get_flags=0): false|array {}
 	public function getMultiByKey(string $server_key, array $keys, int $get_flags=0): false|array {}
-	public function getDelayed(array $keys, bool $with_cas=0, callable $value_cb=NULL): bool {}
-	public function getDelayedByKey(string $server_key, array $keys, bool $with_cas=0, callable $value_cb=NULL): bool {}
+	public function getDelayed(array $keys, bool $with_cas=NULL, callable $value_cb=NULL): bool {}
+	public function getDelayedByKey(string $server_key, array $keys, bool $with_cas=NULL, callable $value_cb=NULL): bool {}
```

Comment:
Seems like `false` would be the correct value then?
https://www.php.net/manual/en/language.types.boolean.php#language.types.boolean.casting

Reply:
You can have a `bool` variable with a `null` default return value. PHP internally changes the type to `bool|null`, the same as it does with the `callable` typehints in this file. The PHP 7 version of the stub file has `null` as the default.

Reply:
PHP will not cast `null` to a bool here. It will keep it as `null`.

Reply:
https://3v4l.org/RlWJe

Reply:
Oh, I see you had FALSE as one of the intermediate commits.